Misc/NEWS.d/next/Core and Builtins/2020-02-11-15-54-40.bpo-39610.fvgsCl.rst

```diff
@@ -0,0 +1,8 @@
+0d ``memoryview`` objects no longer have a ``len``::

```

Comment:
AFAIK there are issues with multi-paragraph NEWS entry.

Reply:
Would you prefer me to keep just the first line here?

Reply:
I am not sure that the meaning of words "have a len()" is clear.

Maybe something like "len() for 0-dimensional memoryview objects raises now a TypeError"?

Reply:
I've changed to your proposed wording.